aura/api_repository.py

```diff
@@ -129,8 +138,12 @@ def make_api_call(method: str, path: str, **kwargs):
     base_url = config.env["base_url"]
```

Comment:
Would it not be better to also accept a `data_api_url` instead of requiring the user to overwrite the `base_url` when they want to switch between managing their db instances vs data apis?

Reply:
I don't think so. This would require a major architectural overhaul of the CLI, which I don't think we have appetite for given this is a Labs project. Additionally, it has been built for a single API future, so I don't think this makes sense for the long term future.

Reply:
Okay, yeah doesn't seem worth it if this is a major architectural change